core-tests/jvm/src/test/scala/zio/ZIOSpecJvm.scala

```diff
@@ -1,767 +1,643 @@
 package zio
```

Comment:
Not sure what's a good way to merge it into `ZIOSpec`.
Right now, it's just a pile of tests with very little grouping by topic/area.
Just dumping everything at end, doesn't sound like a great idea :)

Reply:
Perhaps this should be a separate issue

Reply:
They were originally part of the same file and then got split up in #1947 when some of the tests were migrated, so maybe we can organize it similarly to the original version?

Reply:
Ok, but can we merge this PR and first?
Because I keep seeing changes to `ZIOSpecJvm` in master...
BTW, I'm not sure how to do what you did in #2210 in ZTest.

Reply:
Sure. Let's get this merged.

Reply:
Can you resolve the merge conflict? I think you can just use the `TestClock` in the environment.